jenkins_demo/templates/jobs/stack-os-matrix/config.xml

```diff
@@ -8,7 +8,7 @@
       <parameterDefinitions>
         <hudson.model.StringParameterDefinition>
           <name>BRANCH</name>
-          <description>Whitespace delimited list of branches to attempt to build.  Priority is highest -&gt; lowest from left to right.  &quot;master&apos; is implicitly append to the right side of the list.</description>
+          <description>Whitespace delimited list of &apos;refs&apos; to attempt to build.  Priority is highest -&gt; lowest from left to right.  &quot;master&apos; is implicitly append to the right side of the list, if not specified.</description>
```

Comment:
Indeed.  I dislike the non-DRY nature of these jobs.  I may convert it to be a single job that others call with parameters but I'm concerned about confusing developers further considering the way console output is already 2 tiers deep.  Another option is use a single template in puppet that gets processed into the 3 different jobs.